screens/Signup.js

```diff
@@ -30,108 +30,110 @@ export default function Signup () {
-              placeholder="John Doe"
-              onChangeText={props.handleChange('username')}
-              autoCapitalize="none"
-              // onBlur={handleBlur('username')} what is this
-              value={props.values.username}
-            />
-            <MyTextInput
-              label="Email Address"
-              icon="email-outline"
-              placeholder="abc@gmail.com"
-              onChangeText={props.handleChange('email')}
-              autoCapitalize="none"
-              // onBlur={handleBlur('email')} what is this
-              value={props.values.email}
-            />
-            <PasswordTextInput
-              onChangeText={props.handleChange('password')}
-              // onBlur={handleBlur('password')} what is this
-              autoCapitalize="none"
-              value={props.values.password}
-              secureTextEntry={hidePassword}
-              hidePassword={hidePassword}
-              setHidePassword={setHidePassword}
-            />
-            <PasswordTextInput
-              onChangeText={props.handleChange('confirmpassword')}
-              // onBlur={handleBlur('confirmpassword')} what is this
-              autoCapitalize="none"
-              value={props.values.confirmpassword}
-              secureTextEntry={hidePassword}
-              hidePassword={hidePassword}
-              setHidePassword={setHidePassword}
-            />
-            <Text></Text> 
-            {/* to put space between text and submit */}
-            <BlueButton
-              onPress={props.handleSubmit}
-              buttonText="Submit"
-            />
-          </>
-        )}
+          {(props) => (
+            <>
+              <MyTextInput
+                label="User Name"
+                icon="account-outline"
+                placeholder="John Doe"
+                onChangeText={props.handleChange('username')}
+                autoCapitalize="none"
+                // onBlur={handleBlur('username')} what is this
+                value={props.values.username}
+              />
+              <MyTextInput
+                label="Email Address"
+                icon="email-outline"
+                placeholder="abc@gmail.com"
+                onChangeText={props.handleChange('email')}
+                autoCapitalize="none"
+                // onBlur={handleBlur('email')} what is this
+                value={props.values.email}
+              />
+              <PasswordTextInput
+                Text={<Text>Password</Text>}
+                onChangeText={props.handleChange('password')}
+                // onBlur={handleBlur('password')} what is this
+                autoCapitalize="none"
+                value={props.values.password}
+                secureTextEntry={hidePassword}
+                hidePassword={hidePassword}
+                setHidePassword={setHidePassword}
+              />
+              <PasswordTextInput
+                Text={<Text>Confirm password</Text>}
```

Comment:
here you only have to pass in the text 'Confirm password', given that you wrapped the {Text} with a text component

Reply:
When I tried it didn't work, Something was complaining. That's why I left like this. If you find an answer please change it.